data/human/news.txt

```diff
@@ -798,7 +798,7 @@ news "dentist"
 		word
 			`"`
 		word
-			"Remember to pack a toothcleanser for your journey!"
+			"Remember to pack a toothcleaner for your journey!"
```

Comment:
This change is not a typo. It is supposed to sound unusual.
Neither toothcleanser nor toothcleaner are normal English. Please put it back.